channeldb/channel.go

```diff
@@ -2102,7 +2173,9 @@ func (c *OpenChannel) InsertNextRevocation(revKey *btcec.PublicKey) error {
 // rectify the situation. This method will add the current commitment for the
 // remote party to the revocation log, and promote the current pending
 // commitment to the current remote commitment.
-func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg) error {
+func (c *OpenChannel) AdvanceCommitChainTail(fwdPkg *FwdPkg,
```

Comment:
nit: godoc should be updated to explain the new argument

Reply:
PTAL @ comment